testing/kokoro/system_tests.sh

```diff
@@ -91,8 +91,9 @@ git checkout v2.0.0
 go get -v ./...
 popd
 
-# Always download internal dependencies.
-go get ./internal/...
+# Always download top-level and internal dependencies.
+go get -t ./internal/...
```

Comment:
(I'm not familiar with the repo structure but) looks like `./internal/...` packages are used via subpackage tests (like `./bigquery/snippets`) and not in the root `.`

So, when subpackages are updated, `./internal/...`'s dependencies are going to be captured indirectly into `GO_IMPORTS`.

Even if `./internal/...` was used in root `.`, the `go get -t .` would be sufficient.

tl;dr not sure if this line is necessary.

Reply:
Added this after internal dependencies weren't picked up and it wouldn't compile. So, I think we need it (for one reason or another).